File: 251004/Ryabchikov/tweet-service/src/main/java/by/ryabchikov/tweet_service/dto/creator/CreatorResponseTo.java
package by.ryabchikov.tweet_service.dto.creator;

import lombok.Builder;
import lombok.Data;

import java.io.Serializable;

@Builder
public record CreatorResponseTo (
        Long id,
        String login,
        String password,
        String firstname,
        String lastname
) {
}

File: 251002/Urbanovich/Lab1.Core/Contracts/CreatorResponseTo.cs
using System.Numerics;
namespace Lab1.Core.Contracts
{
    public record CreatorResponseTo
    {
        public long id { get; set; } = 0;
        public string login { get; set; } = string.Empty;
        public string password { get; set; } = string.Empty;
        public string firstname { get; set; } = string.Empty;
        public string lastname { get; set; } = string.Empty;
    }
}
